RazorViewCompilation.sln

```diff
@@ -3,7 +3,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
```

Comment:
upgrade this file to dev15. You'll need to change the project GUID type from 8BB2217D-0F2D-49D1-97BC-3654ED321F3B to 9A19103F-16F7-4668-BE54-9A1E7A4F7556, and update these lines:

```
# Visual Studio 15
VisualStudioVersion = 15.0.26014.0
```